### PR TITLE
Towncrier

### LINKED ISSUE #155: Add a towncrier changelog
### Describe the feature

Need changes

### Proposed solution

_No response_

### PR DESCRIPTION
**Pull request by nabobalis**
Fixes https://github.com/sunpy/streamtracer/issues/155

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -51,9 +51,10 @@ tests = [
   "pytest-xdist",
 ]
 docs = [
-  "sphinx",
-  "sphinx-automodapi",
   "jupyter-sphinx",
+  "sphinx-automodapi",
+  "sphinx-changelog",
+  "sphinx",
```
Comment (Cadair) [inline pyproject.toml]: can we undo this to reduce variance with the template?

File: docs/index.rst [hunk 2]
```diff
@@ -2,9 +2,32 @@
...
+    changelog
+
+Installing
+==========
+
+It is possible to install streamtracer in one go with:
+
+.. code-block:: bash
+
+  pip install streamtracer
```
Comment (Cadair) [inline docs/index.rst:21]:
```suggestion
  pip install streamtracer
  
or using conda with:

.. code-block:: bash
  conda install -c conda-forge streamtracer
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -0,0 +1,89 @@
+2.2.0
+=====
+
+* streamtracer is now faster than the old FORTRAN version expect for -10**3 seeds but every other seed size is faster in Rust now.
```
Comment (Cadair) [inline CHANGELOG.rst]:
```suggestion
* Improved performance of the Rust streamtracing algorithm, so it is now close to or faster than the old FORTRAN version.
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+=====
+
+* streamtracer now includes wheels for Python 3.12, and these have now been uploaded to PyPI as a release.
+* Rust code is now parallelized and while still not as fast as the old FORTRAN version, is near parity.
+
+2.0.1
+=====
+
+* streamtracer now includes wheels for Python 3.11, and these have now been uploaded to PyPI as a release (version 2.0.0 was only ever uploaded as an alpha).
+  streamtracer still does not work in parallel, if you require parallel stream tracing then for now either:
```
Comment (Cadair) [inline CHANGELOG.rst]:
```suggestion
* streamtracer now includes wheels for Python 3.11, and these have now been uploaded to PyPI as a release (version 2.0.0 was only ever uploaded as an alpha).
* streamtracer is now available on conda-forge.
*  streamtracer still does not work in parallel, if you require parallel stream tracing then for now either:
```

File: CHANGELOG.rst [hunk 5]
```diff
@@ -0,0 +1,89 @@
+2.2.0
+=====
+
+* streamtracer is now faster than the old FORTRAN version expect for -10**3 seeds but every other seed size is faster in Rust now.
+
+2.1.0
+=====
+
+* streamtracer now includes wheels for Python 3.12, and these have now been uploaded to PyPI as a release.
```
Comment (Cadair) [inline CHANGELOG.rst]:
```suggestion
* Wheels for Python 3.12 are now built and published on PyPI.
```

File: pyproject.toml (hunk 1 above)
Comment (nabobalis) [reply pyproject.toml]: Yes.

File: CHANGELOG.rst [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+2.0.1
+=====
+
+* streamtracer now includes wheels for Python 3.11, and these have now been uploaded to PyPI as a release (version 2.0.0 was only ever uploaded as an alpha).
+* streamtracer is now available on conda-forge.
+*  streamtracer still does not work in parallel, if you require parallel stream tracing then for now either:
+
+    * Downgrade to version 1.2.0
+    * Manually run several instances of the streamtracer in parallel
```
Comment (nabobalis) [inline CHANGELOG.rst]:
```suggestion
* streamtracer still does not work in parallel, if you require parallel stream tracing then for now either:

    * Downgrade to version 1.2.0
    * Manually run several instances of the streamtracer in parallel
```

**Comment by nabobalis** [comment]
The bot is not following the config:

```
ci:
  autofix_prs: false
```

**Pull request merged**